kickstarter/android-oss
# NT-1128: Lights On card view

# 📲 What

NT-1128: Lights On card view

# 🛠 How
- Added new item_lights_on.xml view
- Added new LigthsOnViewHolder
- filtered by Optimizely feature flag 
- showed just in first tab

# 👀 See
![LightsOn](https://user-images.githubusercontent.com/4083656/81136488-8d3c2800-8f10-11ea-9bc4-90e9ec4b5a94.gif)


# 📋 QA
- ⚠️ make sure your DeviceID is added to optimizely android audience ⚠️ 
- Click on the card, it will open a new activity (don't be scared by the aspect of the opening activity, it Work in Progress at the time this PR is done)
- Check it's visible in just one tab
- Check you have the optimizely feature flag on






# Story 📖

[Lights On Editorial card](https://kickstarter.atlassian.net/browse/NT-1228)
